R/build.R

```diff
@@ -10,9 +10,7 @@
 #'   source. If `NA`, use a built-in list of references to packages pre-modified
 #'   for use with webR. Defaults to `NULL`, meaning no preference over the usual
 #'   remote sources.
-#' @param dependencies Dependency specification for packages to additionally
-#' build. Defaults to `FALSE`, meaning no additional packages. See
-#' [pkgdepends::as_pkg_dependencies] for details.
+#' @inheritParams add_pkg
```

Comment:
@georgestagg I've changed this to inherit the parameter from add_pkg. This will only inherit the parameters that are not documented and will not overwrite anything. Will this work for you?

Reply:
Yeah, that should be OK. We can always remove the inherit later if the function arguments diverge.